Flavour events for bori and amazigh patron deities

Bannana-Puncakes opened the pull request:
Bori and Amazigh patron deities have been added to the existing deity flavour events with appropriate localisations and an additional event picture for Anti.

Commit:
Deities ammended to existing events

Bori and Amazigh patron deities have been added to the existing diety flavour events with appropriate localisations and an additional event picture for Anti.

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -2621,6 +2696,14 @@ character_event = {
+			AND = {
+				religion = a_tag_roog
+				FROM = { religion = a_tag_roog_reformed }
+			}
```
I think we don't need this, because A Tat Roog doesn't have patron deities.

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -3571,6 +3665,16 @@ character_event = {
 	}
+	desc = {
+		trigger = { has_character_modifier = follower_of_samari }
+		text = EVTDESC_105_SAMARI
```
Let's use the vanilla convention and call these localization keys EVTDESC_TOG_7500_SAMARI, etc.  Same with the others.  "105" doesn't mean anything here.

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -4946,6 +5305,8 @@ character_event = {
 				religion = west_african_pagan
 				religion = west_african_pagan_reformed
+				religion = amazigh
+				religion = amazigh_reformed
```
Shouldn't Bori be here too?  Also, you need to put these into the NOR on the next description trigger.

Bannana-Puncakes replied:
Don't think I want Bori here? I was only using the Sun Temple event picture for the Amazigh whilst I think it would be more apt for the Bori to use the shrine event picture?

faiuwle replied:
Oh, yeah, that's fair.  But you still need to add Amazigh to the NOR.

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -5108,6 +5513,8 @@ character_event = {
 				religion = west_african_pagan
 				religion = west_african_pagan_reformed
+				religion = amazigh
+				religion = amazigh_reformed
```
Same here as above.

Bannana-Puncakes replied:
Again same query as above

faiuwle replied:
Similarly here.

faiuwle commented on AfricanReligions/interface/afr_event_pictures.gfx:
```diff
@@ -33,4 +33,9 @@ spriteTypes = {
+    spriteType = {
+    name = "GFX_evt_afr_berber_wrestling"
+    texturefile = "gfx/event_pictures/berber_wrestling.dds"
+    }
```
As Arveanor noted, you aren't indenting these properly.

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -145,3 +145,71 @@ EVTDESC_AFRREL066;The sun lady Magec is beautiful like no other and full of bles
+EVTDESC_106_YAN_RIGA;It is early morning and you are at the shrine dedicated to the Yan Riga, deep in prayers & thought. You hear soft footsteps behind you - who could it be at this hour? [From.GetTitledFirstName], as it turns out. You exchange some pleasantries and soon you find yourself in a full-fledged conversation. You have so much in common!;;;;;;;;;;;;;x
+EVTDESC_106_BABBAKU;Many find it odd that a [Root.GetTitle] works in the fields, but tending to the land and crops make you feel closer to the Babbaku, more so than any prayer. As you take a short break, you notice [From.GetTitledFirstName] approaching. You greet and exchange some words about the harvest. It turns out that [From.GetSheHe] is also a follower of the Babbaku!;;;;;;;;;;;;;x
+EVTDESC_106_TAN_GARKI;It is a fine day and you are out hunting with a small party. As you take a break to eat your midday meal, you find yourself in a discussion on spear throwing with [From.GetTitledFirstName]. It turns out that [From.GetSheHe] is also a devout follower of the Tan Garki. The longer you talk the more you seem to have in common!;;;;;;;;;;;;;x
+EVTDESC_106_SAMARI;Today you have decided to dedicate the night to some gambling and celebration. As a devotee of the Samari you know well that a gambler must put his faith in the Bori and it seems that [From.GetTitledFirstName] agrees. This could be the beginning of a beautiful friendship� as soon as you win the next round.;;;;;;;;;;;;;x
```
I think we've been consistent so far about using lowercase "bori" to talk about the spirits (as opposed to "Bori" to talk about the religion).

Bannana-Puncakes replied:
Ah wasn't aware. I'll make sure to maintain this.

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -145,3 +145,71 @@ EVTDESC_AFRREL066;The sun lady Magec is beautiful like no other and full of bles
+EVTDESC_106_BABBAKU;Many find it odd that a [Root.GetTitle] works in the fields, but tending to the land and crops make you feel closer to the Babbaku, more so than any prayer. As you take a short break, you notice [From.GetTitledFirstName] approaching. You greet and exchange some words about the harvest. It turns out that [From.GetSheHe] is also a follower of the Babbaku!;;;;;;;;;;;;;x
+EVTDESC_106_TAN_GARKI;It is a fine day and you are out hunting with a small party. As you take a break to eat your midday meal, you find yourself in a discussion on spear throwing with [From.GetTitledFirstName]. It turns out that [From.GetSheHe] is also a devout follower of the Tan Garki. The longer you talk the more you seem to have in common!;;;;;;;;;;;;;x
+EVTDESC_106_SAMARI;Today you have decided to dedicate the night to some gambling and celebration. As a devotee of the Samari you know well that a gambler must put his faith in the Bori and it seems that [From.GetTitledFirstName] agrees. This could be the beginning of a beautiful friendship� as soon as you win the next round.;;;;;;;;;;;;;x
+EVTDESC_106_YAYAN_ZANZANNA;As a good ruler you decide to visit the ill being attended by the local healer. When you arrive, you see that [From.GetTitledFirstName] is already there. After some talking you realize that [From.GetSheHe] is also a devotee of the Yayan zanzanna, which is a blessing  that you are surrounded by so much disease.;;;;;;;;;;;;;x
```
I'm not sure the last sentence makes sense.  Can you ask for it to be rephrased?

Bannana-Puncakes replied:
Don't know how but I had deleted a word is it OK now?

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -145,3 +145,71 @@ EVTDESC_AFRREL066;The sun lady Magec is beautiful like no other and full of bles
+EVTDESC_106_TANIT;It is early morning and you are at the shrine dedicated to Tanit, deep in prayers. You hear soft footsteps behind you - who could it be at this hour? [From.GetTitledFirstName], as it turns out. You exchange some pleasantries and soon you find yourself in a full-fledged conversation. You have so much in common!;;;;;;;;;;;;;x
+EVTDESC_106_ANTI;Today [From.GetTitledFirstName] has invited you to watch a wrestling match between two local champions, after the first round of the competition you have had time to talk and soon discovered that [From.GetSheHe] is also a devotee of �nti, the great giant fighter.;;;;;;;;;;;;;x
+EVTDESC_106_GURZIL;It is a fine day and you are out hunting with a small party. As you take a break to eat your midday meal, you find yourself in a discussion on archery with [From.GetTitledFirstName]. It turns out that [From.GetSheHe] is also a devout follower of �nti. The longer you talk the more you seem to have in common!;;;;;;;;;;;;;x
+EVTDESC_106_MAGEC;It is said that the nobles are pale because they spend all their time indoors, but not you. As a devotee of Magec the sun you love walking through the fields enjoying its light. On one of your walks you meet [From.GetTitledFirstName] who is also dedicating his day to Magec. It is always good to find another devotee.;;;;;;;;;;;;;x
```
Needs to be `[From.GetHerHis]` instead of "his".

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -145,3 +145,71 @@ EVTDESC_AFRREL066;The sun lady Magec is beautiful like no other and full of bles
+EVTDESC_110_SAMARI;In your efforts to become more like the good Samari, you have begun treating everyone who crosses your path with more openness and respect. As a result, you have become known for your compassionate ways. Some go as far as calling you "the Voice of the Samari".;;;;;;;;;;;;;x
+EVTDESC_110_TANIT;In your efforts to become more like the good Tanit, you have begun treating everyone who crosses your path with more understanding and respect. As a result, you have become known for your compassionate ways and noble nature. Some go as far as calling you "the Voice of Tanit".;;;;;;;;;;;;;x
+EVTOPTA_110_SAMARI;I am proud to honour the Samari.;;;;;;;;;;;;;x
+EVTDESC_110_BABBAKU;Your efforts to make [Root.Capital.GetName] famous for its markets and merchants has paid of. The city is prospering like never before, and more people gather at the shrine of the Samari each day. You have inspired many to follow his path.;;;;;;;;;;;;;x
```
"Their" instead of "his", probably?

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -145,3 +145,71 @@ EVTDESC_AFRREL066;The sun lady Magec is beautiful like no other and full of bles
+EVTDESC_110_SET_ME;Following in the footsteps of Set has brought you down many paths you would never otherwise tread. The things one can overhear during such journeys often prove valuable, indeed. Just recently you discovered that [god_plotter.GetTitledFirstName] is plotting against none other than yourself!;;;;;;;;;;;;;x
+EVTDESC_110_YAYAN_ZANZANNA;Following in the footsteps of the Yayan Zanzanna has brought you down many paths you would never otherwise tread. The things one can overhear during such journeys often prove valuable, indeed. Just recently you discovered that [god_plotter.GetTitledFirstName] is plotting against [god_plot_target.GetTitledFirstName]...;;;;;;;;;;;;;x
+EVTDESC_110_SET;Following in the footsteps of Set has brought you down many paths you would never otherwise tread. The things one can overhear during such journeys often prove valuable, indeed. Just recently you discovered that [god_plotter.GetTitledFirstName] is plotting against [god_plot_target.GetTitledFirstName]...;;;;;;;;;;;;;x
+EVTDESC_110_YAN_RIGA;In your search for wisdom in the name of the Yan Riga, you have had more than one epiphany relevant to welfare of your lands. Spreading that knowledge is not only beneficial to your rule, but also a sacred duty in and of itself.In your search for wisdom in the name of Tsumburbura, you have had more than one epiphany relevant to welfare of your lands. Spreading that knowledge  ...
```
You seem to have copied one of the other event descriptions.

faiuwle reviewed (changes requested):
You're missing a number of deities for Theme set 1 - I count Yan Riga, Babbaku, Tan Garki, Änti, Gurzil, Magec, and Set with no event for that set.  You don't have to just use the vanilla events, you can get TheDude to write custom events for each one.  I also didn't seem to see event localizations to go with at least some of the Theme set 1 events, but it's hard to tell due to how confusingly you named the localization keys.

ghost commented:
I'll sort out the localisation keys, made sense in my head but yeah quite confusing. As to Theme one, the vanilla themes don't work for all of our deities, I talked to The Dude and the ones I've done where what we agreed originally. I can definitely do new events but they weren't a necessity and when I asked a couple days back. Do you want all the deities to have their own theme one flavour event? I'll talk to The Dude and get it sorted.

Commit:
PR changes 1

faiuwle commented:
Yeah, I want them to all have one event for each random outcome, I think I said that in that past.

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -4723,6 +5038,15 @@ character_event = {
+	}
+	desc = {
+		trigger = { has_character_modifier = follower_of_magec }
+		text = EVTDESC_TOG_7520_MAGEC
```
Did you mean for there to be a picture here?

Bannana-Puncakes replied:
No. I was going to use the default picture for that event, the tribal lands picture.

faiuwle replied:
Does that work?  Or do you have to specify that picture here because you specified the picture in the other `desc` blocks?

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -145,3 +145,71 @@ EVTDESC_AFRREL066;The sun lady Magec is beautiful like no other and full of bles
+EVTDESC_TOG_7506_TANIT;It is early morning and you are at the shrine dedicated to Tanit, deep in prayers. You hear soft footsteps behind you - who could it be at this hour? [From.GetTitledFirstName], as it turns out. You exchange some pleasantries and soon you find yourself in a full-fledged conversation. You have so much in common!;;;;;;;;;;;;;x
+EVTDESC_TOG_7506_ANTI;Today [From.GetTitledFirstName] has invited you to watch a wrestling match between two local champions, after the first round of the competition you have had time to talk and soon discovered that [From.GetSheHe] is also a devotee of �nti, the great giant fighter.;;;;;;;;;;;;;x
+EVTDESC_TOG_7506_GURZIL;It is a fine day and you are out hunting with a small party. As you take a break to eat your midday meal, you find yourself in a discussion on archery with [From.GetTitledFirstName]. It turns out that [From.GetSheHe] is also a devout follower of �nti. The longer you talk the more you seem to have in common!;;;;;;;;;;;;;x
+EVTDESC_TOG_7506_MAGEC;It is said that the nobles are pale because they spend all their time indoors, but not you. As a devotee of Magec the sun you love walking through the fields enjoying its light. On one of your walks you meet [From.GetTitledFirstName] who is also dedicating [From.GetHisHer] day to Magec. It is always good to find another devotee.;;;;;;;;;;;;;x
```
It's `GetHerHis`.

Commit:
Final commit

All changes made and validated and game tested.

Commit:
Removed my copy folder...

Well oops

Commit:
Today I learn to spell

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -2649,6 +2772,17 @@ character_event = {
+		remove_character_modifier = follower_of_anti
+		remove_character_modifier = follower_of_gurzil
+		remove_character_modifier = follower_of_magec
+		remove_character_modifier = follower_of_set
```
Shouldn't we also remove the other modifiers that you add in these events?

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -4571,6 +4832,295 @@ character_event = {
+	picture = GFX_evt_market_byzantine
+
+	desc = {
+		trigger = { has_character_modifier = follower_of_yan_riga }
```
There's no need for a trigger or anything here.

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -4571,6 +4832,295 @@ character_event = {
 
+#Yan Riga flavour event
+character_event = {
+	id = TOG.7530
```
Let's create these in the AFRREL namespace, in their own file, so we don't risk incompatibility with the game.

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -4571,6 +4832,295 @@ character_event = {
+	is_triggered_only = yes
+
+	option = {
+		name = { text = EVTOPTA_TOG_7530 }
```
`name = key` is fine for these.

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -4571,6 +4832,295 @@ character_event = {
+
+	option = {
+		name = { text = EVTOPTA_TOG_7532 }
+		remove_trait = is_fat
```
You can't just remove these traits, they'll get added back as soon as the game checks the character's weight again.  You have to actually modify the variable that determines the character's weight.  Look in the scripted effects for weight gain and loss for that.

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -4571,6 +4832,295 @@ character_event = {
+	border = GFX_event_normal_frame_religion
+	picture = GFX_evt_magic_ritual
+
+	desc = {
```
Doesn't need this whole syntax here.

Bannana-Puncakes replied:
What do you mean?

faiuwle replied:
You can just say `desc = localization_key` if there aren't multiple descriptions.

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -4571,6 +4832,295 @@ character_event = {
+	is_triggered_only = yes
+
+	option = {
+		name = { text = EVTOPTA_TOG_7533 }
```
Same

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -4571,6 +4832,295 @@ character_event = {
+	border = GFX_event_normal_frame_religion
+	picture = GFX_evt_battle_scene
+
+	desc = {
```
Same

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -4571,6 +4832,295 @@ character_event = {
+	is_triggered_only = yes
+
+	option = {
+		name = { text = EVTOPTA_TOG_7534 }
```
You get the idea

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -4571,6 +4832,295 @@ character_event = {
+				OR = {
+					has_disease_trigger = yes
+					trait = incapable
+				}
```
You don't need to check for the modifier here, but if you're going to check for disease or incapable, you need to have a description that you get when neither of those things are true.

Bannana-Puncakes replied:
I've set it to check in the TOG.7100 so the event shouldn't be called if the character is healthy

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -4571,6 +4832,295 @@ character_event = {
+		name = { text = EVTOPTA_TOG_7536 }
+		add_character_modifier = {
+			name = blessing_of_ayu
+			duration = -1
```
Shouldn't this have a duration of 5 years like the other ones?

Bannana-Puncakes replied:
The spreadsheet had nothing about duration so I went with that. Can change it if that's whats wanted?

Bannana-Puncakes replied:
5 years for all of them?

faiuwle replied:
It doesn't make sense to make it permanent.  Just give it a duration based on what you think is reasonable, based on the bonuses.

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -4571,6 +4832,295 @@ character_event = {
+		name = { text = EVTOPTB_TOG_7536 }
+		add_character_modifier = {
+			name = blessing_of_kadda
+			duration = -1
```
Same

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -4571,6 +4832,295 @@ character_event = {
+	desc = {
+		trigger = { 
+			has_character_modifier = follower_of_anti
+			is_married = yes
```
Again, if this one is only for married people, you have to specify a description for unmarried people too.

Bannana-Puncakes replied:
Done the same as for Yayan Zanzanna

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -4571,6 +4832,295 @@ character_event = {
+			}
+		}
+		name = { text = EVTOPTA_TOG_7537 }
+		remove_trait = is_fat
```
Again, you can't just remove this.

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -4571,6 +4832,295 @@ character_event = {
+			chance = 50
+			save_event_target_as = self
+			random_spouse = {
+				add_lover = event_target:self
```
You don't need to save an event target here, just say `add_lover = PREV`.

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -4571,6 +4832,295 @@ character_event = {
+	}
+	option = {
+		name = { text = EVTOPTB_TOG_7537 }
+		remove_trait = is_fat
```
Again

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -4571,6 +4832,295 @@ character_event = {
+		gold = -10
+		add_character_modifier = {
+			name = blessing_of_gurzil
+			duration = -1
```
Again, should have a finite duration.

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -145,3 +145,95 @@ EVTDESC_AFRREL066;The sun lady Magec is beautiful like no other and full of bles
+EVTDESC_TOG_7506_BABBAKU;Many find it odd that a [Root.GetTitle] works in the fields, but tending to the land and crops make you feel closer to the Babbaku, more so than any prayer. As you take a short break, you notice [From.GetTitledFirstName] approaching. You greet and exchange some words about the harvest. It turns out that [From.GetSheHe] is also a follower of the Babbaku!;;;;;;;;;;;;;x
+EVTDESC_TOG_7506_TAN_GARKI;It is a fine day and you are out hunting with a small party. As you take a break to eat your midday meal, you find yourself in a discussion on spear throwing with [From.GetTitledFirstName]. It turns out that [From.GetSheHe] is also a devout follower of the Tan Garki. The longer you talk the more you seem to have in common!;;;;;;;;;;;;;x
+EVTDESC_TOG_7506_SAMARI;Today you have decided to dedicate the night to some gambling and celebration. As a devotee of the Samari you know well that a gambler must put his faith in the bori and it seems that [From.GetTitledFirstName] agrees. This could be the beginning of a beautiful friendship� as soon as you win the next round.;;;;;;;;;;;;;x
+EVTDESC_TOG_7506_YAYAN_ZANZANNA;As a good ruler you decide to visit the ill being attended by the local healer. When you arrive, you see that [From.GetTitledFirstName] is already there. After some talking you realize that [From.GetSheHe] is also a devotee of the Yayan zanzanna, which is a blessing given that you are surrounded by so much disease.;;;;;;;;;;;;;x
```
Zanzanna maybe should be capitalized?

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -145,3 +145,95 @@ EVTDESC_AFRREL066;The sun lady Magec is beautiful like no other and full of bles
+EVTDESC_TOG_7506_SET;When you needed an accomplice in a small scheme, [From.GetTitledFirstName] proved to be an unexpected ally. Think what mutual benefits can do to unite two souls! While it was pure business at first, it turns out you have plenty in common, such as your reverence of Set. Now you spend more time conversing than plotting...;;;;;;;;;;;;;x
+EVTDESC_TOG_7510_SAMARI;You are dancing at the secret rite held in the Samari's honor as your eyes find [From.GetTitledFirstName]'s across the bonfire. For a moment, it is as if the whole world turns silent. The drumming is a distant thunder, yet the rhythmic vibrations run through your body, to your very core...;;;;;;;;;;;;;x
+EVTOPTA_107_SAMARI;Tonight, I shall bed [From.GetFirstName] in the name of the Samari.;;;;;;;;;;;;;x
+EVTOPTA_TOG_7510_SAMARI;Lately, you have striven to act in the spirit of Tanit by treating your [From.GetHusbandWife] [From.GetTitledFirstName] with respect and warmth. Slowly it has changed the dynamic of your relationship, and those who are closest to you have noticed it as well.;;;;;;;;;;;;;x
```
Looks like you mistakenly put that key here instead, when this text is about Tanit.

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -145,3 +145,95 @@ EVTDESC_AFRREL066;The sun lady Magec is beautiful like no other and full of bles
+EVTDESC_TOG_7506_MAGEC;It is said that the nobles are pale because they spend all their time indoors, but not you. As a devotee of Magec the sun you love walking through the fields enjoying its light. On one of your walks you meet [From.GetTitledFirstName] who is also dedicating [From.GetHerHis] day to Magec. It is always good to find another devotee.;;;;;;;;;;;;;x
+EVTDESC_TOG_7506_SET;When you needed an accomplice in a small scheme, [From.GetTitledFirstName] proved to be an unexpected ally. Think what mutual benefits can do to unite two souls! While it was pure business at first, it turns out you have plenty in common, such as your reverence of Set. Now you spend more time conversing than plotting...;;;;;;;;;;;;;x
+EVTDESC_TOG_7510_SAMARI;You are dancing at the secret rite held in the Samari's honor as your eyes find [From.GetTitledFirstName]'s across the bonfire. For a moment, it is as if the whole world turns silent. The drumming is a distant thunder, yet the rhythmic vibrations run through your body, to your very core...;;;;;;;;;;;;;x
+EVTOPTA_107_SAMARI;Tonight, I shall bed [From.GetFirstName] in the name of the Samari.;;;;;;;;;;;;;x
```
Shouldn't this be `EVTOPTA_7510_SAMARI`?

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -145,3 +145,95 @@ EVTDESC_AFRREL066;The sun lady Magec is beautiful like no other and full of bles
+EVTOPTA_TOG_7516_AMAZIGH;Glory to the great god!;;;;;;;;;;;;;x
+EVTDESC_TOG_7518_SAMARI;In your efforts to become more like the good Samari, you have begun treating everyone who crosses your path with more openness and respect. As a result, you have become known for your compassionate ways. Some go as far as calling you "the Voice of the Samari".;;;;;;;;;;;;;x
+EVTDESC_TOG_7518_TANIT;In your efforts to become more like the good Tanit, you have begun treating everyone who crosses your path with more openness and respect. As a result, you have become known for your compassionate ways. Some go as far as calling you "the Voice of Tanit".;;;;;;;;;;;;;x
+EVTDESC_TOG_7510_TANIT;In your efforts to become more like the good Tanit, you have begun treating everyone who crosses your path with more understanding and respect. As a result, you have become known for your compassionate ways and noble nature. Some go as far as calling you "the Voice of Tanit".;;;;;;;;;;;;;x
```
Duplicate with wrong key?

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -145,3 +145,95 @@ EVTDESC_AFRREL066;The sun lady Magec is beautiful like no other and full of bles
+EVTDESC_TOG_7518_TANIT;In your efforts to become more like the good Tanit, you have begun treating everyone who crosses your path with more openness and respect. As a result, you have become known for your compassionate ways. Some go as far as calling you "the Voice of Tanit".;;;;;;;;;;;;;x
+EVTDESC_TOG_7510_TANIT;In your efforts to become more like the good Tanit, you have begun treating everyone who crosses your path with more understanding and respect. As a result, you have become known for your compassionate ways and noble nature. Some go as far as calling you "the Voice of Tanit".;;;;;;;;;;;;;x
+EVTOPTA_TOG_7518_SAMARI;I am proud to honour the Samari.;;;;;;;;;;;;;x
+EVTDESC_TOG_7520_BABBAKU;Your efforts to make [Root.Capital.GetName] famous for its markets and merchants has paid of. The city is prospering like never before, and more people gather at the shrine of the Samari each day. You have inspired many to follow their path.;;;;;;;;;;;;;x
```
Samari != Babbaku

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -145,3 +145,95 @@ EVTDESC_AFRREL066;The sun lady Magec is beautiful like no other and full of bles
+EVTDESC_TOG_7522_YAYAN_ZANZANNA_ME;Following in the footsteps of The Yayan Zanzanna has brought you down many paths you would never otherwise tread. The things one can overhear during such journeys often prove valuable, indeed. Just recently you discovered that [god_plotter.GetTitledFirstName] is plotting against none other than yourself!;;;;;;;;;;;;;x
+EVTDESC_TOG_7522_SET_ME;Following in the footsteps of Set has brought you down many paths you would never otherwise tread. The things one can overhear during such journeys often prove valuable, indeed. Just recently you discovered that [god_plotter.GetTitledFirstName] is plotting against none other than yourself!;;;;;;;;;;;;;x
+EVTDESC_TOG_7522_YAYAN_ZANZANNA;Following in the footsteps of the Yayan Zanzanna has brought you down many paths you would never otherwise tread. The things one can overhear during such journeys often prove valuable, indeed. Just recently you discovered that [god_plotter.GetTitledFirstName] is plotting against [god_plot_target.GetTitledFirstName]...;;;;;;;;;;;;;x
+EVTDESC_TOG_7522_SET;Following in the footsteps of Set has brought you down many paths you would never otherwise tread. The things one can overhear during such journeyEVTDESC_TOG_7522_SETs often prove valuable, indeed. Just recently you discovered that [god_plotter.GetTitledFirstName] is plotting against [god_plot_target.GetTitledFirstName]...;;;;;;;;;;;;;x
```
You pasted the localization key into the middle of this text.

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -145,3 +145,95 @@ EVTDESC_AFRREL066;The sun lady Magec is beautiful like no other and full of bles
+EVTDESC_TOG_7522_SET_ME;Following in the footsteps of Set has brought you down many paths you would never otherwise tread. The things one can overhear during such journeys often prove valuable, indeed. Just recently you discovered that [god_plotter.GetTitledFirstName] is plotting against none other than yourself!;;;;;;;;;;;;;x
+EVTDESC_TOG_7522_YAYAN_ZANZANNA;Following in the footsteps of the Yayan Zanzanna has brought you down many paths you would never otherwise tread. The things one can overhear during such journeys often prove valuable, indeed. Just recently you discovered that [god_plotter.GetTitledFirstName] is plotting against [god_plot_target.GetTitledFirstName]...;;;;;;;;;;;;;x
+EVTDESC_TOG_7522_SET;Following in the footsteps of Set has brought you down many paths you would never otherwise tread. The things one can overhear during such journeyEVTDESC_TOG_7522_SETs often prove valuable, indeed. Just recently you discovered that [god_plotter.GetTitledFirstName] is plotting against [god_plot_target.GetTitledFirstName]...;;;;;;;;;;;;;x
+EVTDESC_TOG_7524_YAN_RIGA;In your search for wisdom in the name of the Yan Riga, you have had more than one epiphany relevant to the 	welfare of your lands. Spreading that knowledge is not only beneficial to your rule, but also a sacred duty in and of itself.;;;;;;;;;;;;;x
```
Random space in the middle?

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -145,3 +145,95 @@ EVTDESC_AFRREL066;The sun lady Magec is beautiful like no other and full of bles
+EVTDESC_TOG_7536;Ayu the manatee is one of the Tan Dowa wild bori, he drags his victims underwater and drowns them. Kadda the crocodile is a lot more direct, as he break them with his jaws. In a recent spirit possession I had the opportunity to choose which of these spirits to call.;;;;;;;;;;;;;x
+EVTOPTA_TOG_7536;The sneaky manatee.;;;;;;;;;;;;;x
+EVTOPTB_TOG_7536;The ferocious crocodile.;;;;;;;;;;;;;x
+EVTDESC_TOG_7537;Recently I have being practicing a lot of wrestling, as the giant �nti did. It not only has made me feel great but it seems like my new defined abdomen has caught the eye of my [From.GetHusbandWife].;;;;;;;;;;;;;x
```
I checked this one, and `FROM` is just the same as `ROOT` here.  Can you go back through your events and check that the localization functions actually make sense?

Bannana-Puncakes replied:
Nope they're the wrong way round. Should I be using PREV here?

faiuwle replied:
No, PREV doesn't make any sense here.  You have to set up the event so that FROM is the right person, like the lover and new friend events do.  You need to read about how scopes work on the wiki if you don't understand them.

Commit:
Kestrellyn help

All changes made except the file movement problem and the Anti husband/wife localisation (can't check it properly until file problem resolved)

Commit:
Final Changes

Localisations and Anti event fixed

faiuwle commented on AfricanReligions/common/event_modifiers/afr_event_modifiers.txt:
```diff
@@ -1,4 +1,4 @@
-secret_bori_community = {
+	secret_bori_community = {
```
Can this change be removed?

faiuwle commented on AfricanReligions/events/afr_amazigh_patron_deity_events.txt:
```diff
@@ -194,4 +194,132 @@ character_event = {
+    is_triggered_only = yes
+ 
+    option = {
+        trigger = {
```
If this trigger fails, there will just be an event that can't be exited because it doesn't have any options.  Take the triggers out of here and simply ensure that the spouse who launches this event is alive and not in prison in the first place.

Edit:  I see you do that below.  In that case, these triggers are unnecessary.

faiuwle commented on AfricanReligions/events/afr_amazigh_patron_deity_events.txt:
```diff
@@ -194,4 +194,132 @@ character_event = {
+        }
+        random = {
+            chance = 50
+            random_spouse = {
```
`random_spouse` is not necessarily the spouse who launched the event.  You should use `FROM` instead.

faiuwle commented on AfricanReligions/events/afr_amazigh_patron_deity_events.txt:
```diff
@@ -194,4 +194,132 @@ character_event = {
+        gold = -10
+        add_character_modifier = {
+            name = blessing_of_gurzil
+            duration = 5
```
I think this comes out to 5 days.  Did you mean to make it `years = 5`?

faiuwle commented on AfricanReligions/events/afr_bori_patron_deity_events.txt:
```diff
@@ -210,4 +210,153 @@ character_event = {
+        name = EVTOPTA_AFRREL_112
+        add_character_modifier = {
+            name = blessing_of_ayu
+            duration = 5
```
Same duration note from above.

faiuwle commented on AfricanReligions/events/afr_bori_patron_deity_events.txt:
```diff
@@ -210,4 +210,153 @@ character_event = {
+        name = EVTOPTB_AFRREL_112
+        add_character_modifier = {
+            name = blessing_of_kadda
+            duration = 5
```
Same duration note from above.

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -2509,9 +2567,88 @@ character_event = {
+								is_female = no
+								AND = {
+									is_female = yes
+									NOT = { age = 46 }
```
Why this age restriction?

Bannana-Puncakes replied:
The vanilla events I looked at had this restriction for most romantic/sexual events for women. I don't see the point I just kept it for consistency

faiuwle replied:
It's probably because there's a chance or a guarantee of pregnancy being scripted.  Since that's not the case for this one, I don't think there should be an age restriction.

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -2509,9 +2567,88 @@ character_event = {
+									NOT = { age = 46 }
+								}
+							}
+							NOR = {
```
Should probably make these checks for ROOT, too.

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -145,3 +145,95 @@ EVTDESC_AFRREL066;The sun lady Magec is beautiful like no other and full of bles
+EVTDESC_TOG_7506_SET;When you needed an accomplice in a small scheme, [From.GetTitledFirstName] proved to be an unexpected ally. Think what mutual benefits can do to unite two souls! While it was pure business at first, it turns out you have plenty in common, such as your reverence of Set. Now you spend more time conversing than plotting...;;;;;;;;;;;;;x
+EVTDESC_TOG_7510_SAMARI;You are dancing at the secret rite held in the Samari's honor as your eyes find [From.GetTitledFirstName]'s across the bonfire. For a moment, it is as if the whole world turns silent. The drumming is a distant thunder, yet the rhythmic vibrations run through your body, to your very core...;;;;;;;;;;;;;x
+EVTOPTA_TOG_7510_SAMARI;Tonight, I shall bed [From.GetFirstName] in the name of the Samari.;;;;;;;;;;;;;x
+EVTDESC_TOG_7510_TANIT;Lately, you have striven to act in the spirit of Tanit by treating your [From.GetHusbandWife] [From.GetTitledFirstName] with respect and warmth. Slowly it has changed the dynamic of your relationship, and those who are closest to you have noticed it as well.;;;;;;;;;;;;;x
```
Surely it should just be `[From.GetFirstName]` here, right?

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -145,3 +145,95 @@ EVTDESC_AFRREL066;The sun lady Magec is beautiful like no other and full of bles
+EVTDESC_TOG_7518_SAMARI;In your efforts to become more like the good Samari, you have begun treating everyone who crosses your path with more openness and respect. As a result, you have become known for your compassionate ways. Some go as far as calling you "the Voice of the Samari".;;;;;;;;;;;;;x
+EVTDESC_TOG_7518_TANIT;In your efforts to become more like the good Tanit, you have begun treating everyone who crosses your path with more openness and respect. As a result, you have become known for your compassionate ways. Some go as far as calling you "the Voice of Tanit".;;;;;;;;;;;;;x
+EVTOPTA_TOG_7518_SAMARI;I am proud to honour the Samari.;;;;;;;;;;;;;x
+EVTDESC_TOG_7520_BABBAKU;Your efforts to make [Root.Capital.GetName] famous for its markets and merchants has paid of. The city is prospering like never before, and more people gather at the shrine of the Babbaku each day. You have inspired many to follow their path.;;;;;;;;;;;;;x
```
"paid of" -> "paid off"

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -145,3 +145,95 @@ EVTDESC_AFRREL066;The sun lady Magec is beautiful like no other and full of bles
+EVTOPTA_AFRREL_110;One, two, one, two;;;;;;;;;;;;;x
+EVTDESC_AFRREL_111;As my illness worsens I have tried contacting with the bori spirits. Today my prayers have being answered as I felt how I was possessed by KaiKai, one of the Yayan Zanzanna children of the plague. The lady of the disease took it away, my pain is finally gone.;;;;;;;;;;;;;x
+EVTOPTA_AFRREL_111;I must sacrifice two hens in her name.;;;;;;;;;;;;;x
+EVTOPTB_AFRREL_111;I have no need of her.;;;;;;;;;;;;;x
```
This doesn't appear to be used anymore.

faiuwle commented on AfricanReligions/events/afr_bori_patron_deity_events.txt:
```diff
@@ -210,4 +210,153 @@ character_event = {
+        name = EVTOPTA_AFRREL_107
+        random_list = {
+            50 = { hidden_tooltip = { character_event = { id = AFRREL.108 days = 3 } } }
+            50 = { hidden_tooltip = { character_event = { id = AFRREL.109 days = 3 } } }
```
You need to either put `hidden_tooltip` around the entire `random_list` block, or provide appropriate tooltips for the events.

Bannana-Puncakes replied:
Do you mean:
hidden_tooltip = {
            random_list = {
                50 = { character_event = { id = AFRREL.108 days = 3 } }
                50 = {  character_event = { id = AFRREL.109 days = 3 } }
            }
        }

faiuwle replied:
Right.

faiuwle commented on AfricanReligions/events/afr_bori_patron_deity_events.txt:
```diff
@@ -210,4 +210,153 @@ character_event = {
+character_event = {
+    id = AFRREL.110
+    border = GFX_event_normal_frame_religion
+    picture = GFX_evt_battle_scene
```
This picture isn't showing up.

faiuwle commented on AfricanReligions/events/afr_bori_patron_deity_events.txt:
```diff
@@ -210,4 +210,153 @@ character_event = {
+character_event = {
+    id = AFRREL.111
+    border = GFX_event_normal_frame_religion
+    picture = GFX_evt_sick_character
```
This picture isn't showing up.

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -2509,9 +2567,88 @@ character_event = {
+					limit = { 
+						OR = {
+							has_character_modifier = follower_of_taipo_and_mielikki 
+							has_character_modifier = follower_of_tan_dowa
```
This causes the unrelated event for Taipo and Mielikki to fire if you have the Tan Dowa modifier.

faiuwle commented on AfricanReligions/events/afr_amazigh_patron_deity_events.txt:
```diff
@@ -194,4 +194,132 @@ character_event = {
+
+    option = {
+        name = EVTOPTA_AFRREL_114
+        gold = -10
```
It's `wealth`, not "gold".

faiuwle commented on AfricanReligions/events/afr_bori_patron_deity_events.txt:
```diff
@@ -210,4 +210,153 @@ character_event = {
+
+    option = {
+        name = EVTOPTA_AFRREL_111
+        gold = -10
```
Should be `wealth`.

faiuwle commented on AfricanReligions/events/oldgods_ancestor_worship_events.txt:
```diff
@@ -4908,6 +5316,10 @@ character_event = {
+	desc = {
+		trigger = { has_character_modifier = follower_of_yan_riga }
+		text = EVTDESC_TOG_7524_YAN_RIGA
+	}
```
You also need to relocalize the option text, which inappropriately mentions a goddess.

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -145,3 +145,95 @@ EVTDESC_AFRREL066;The sun lady Magec is beautiful like no other and full of bles
+EVTDESC_TOG_7518_TANIT;In your efforts to become more like the good Tanit, you have begun treating everyone who crosses your path with more openness and respect. As a result, you have become known for your compassionate ways. Some go as far as calling you "the Voice of Tanit".;;;;;;;;;;;;;x
+EVTOPTA_TOG_7518_SAMARI;I am proud to honour the Samari.;;;;;;;;;;;;;x
+EVTDESC_TOG_7520_BABBAKU;Your efforts to make [Root.Capital.GetName] famous for its markets and merchants has paid of. The city is prospering like never before, and more people gather at the shrine of the Babbaku each day. You have inspired many to follow their path.;;;;;;;;;;;;;x
+EVTDESC_TOG_7520_MAGEC;Your efforts to make the most out Magec's holy soil has turned you into a true inspiration for your people. Their hard work, in combination with the fair weather which the Goddess has blessed you with, has resulted in the richest harvest in many years. Celebrations in Magec's honor is held in every village.;;;;;;;;;;;;;x
```
Needs an "of" after "out".

faiuwle commented on AfricanReligions/localisation/afr_event_texts.csv:
```diff
@@ -145,3 +145,95 @@ EVTDESC_AFRREL066;The sun lady Magec is beautiful like no other and full of bles
+EVTDESC_AFRREL_111;As my illness worsens I have tried contacting with the bori spirits. Today my prayers have being answered as I felt how I was possessed by KaiKai, one of the Yayan Zanzanna children of the plague. The lady of the disease took it away, my pain is finally gone.;;;;;;;;;;;;;x
+EVTOPTA_AFRREL_111;I must sacrifice two hens in her name.;;;;;;;;;;;;;x
+EVTOPTB_AFRREL_111;I have no need of her.;;;;;;;;;;;;;x
+EVTDESC_AFRREL_112;Ayu the manatee is one of the Tan Dowa wild bori, he drags his victims underwater and drowns them. Kadda the crocodile is a lot more direct, as he break them with his jaws. In a recent spirit possession I had the opportunity to choose which of these spirits to call.;;;;;;;;;;;;;x
```
"break" -> "breaks"

faiuwle reviewed (changes requested):
Ok, I've tested everything except the monument, but I think that's probably fine.

Commit:
What I managed of the review

Sorry this ain't everything

faiuwle committed:
Remaining fixes on PR
